consent/strategy_logout_test.go

```diff
@@ -238,7 +257,7 @@ func TestLogoutFlows(t *testing.T) {
 
 	t.Run("case=should execute backchannel logout if issued without rp-involvement", func(t *testing.T) {
 		sid := make(chan string)
-		acceptLoginAsAndWatchSid(t, subject, sid)
+		acceptLoginAsAndWatchSid(t, subject, sid, 1)
```

Comment:
Honestly, not a big fan of this approach, because it is confusing. AFAIK, golang does not have default argument values to skip this change.

Reply:
Fixed with a wrapper function to encapsulate this logic for most tests